src/workspaceViewer.ts

```diff
@@ -84,24 +88,33 @@ export class WorkspaceItem extends TreeItem {
 		str: string,
 		type: string,
 		length: number,
-		collapsibleState: TreeItemCollapsibleState
+		collapsibleState: TreeItemCollapsibleState,
+		dim?: number[]
 	) {
 		super(label, collapsibleState);
-		this.description = str;
+		this.description = this.getDescription(dim, str);
 		this.tooltip = `${label} (${rClass}, length of ${length})`;
 		this.contextValue = type;
 	}
+
+	private getDescription(dim: number[], str: string): string {
+		if (dim !== undefined) {
+			return `${dim[0]} obs. of ${dim[1]} variables`
```

Comment:
Is it better to show the first class of the object such that 

```
data.table: ? objs. of ? variables
```

Also, we need to make it right for one variable case by using "variable".